Assignment/ComplexTest.c

```diff
@@ -0,0 +1,50 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Vinitha Perumal - GET
+// ------------------------------------------------------------------
+// ComplexTest.c
+// Program on Test1.2 branch.
+// ------------------------------------------------------------------------------------------------
+#include "Complex.h"
+#include <stdio.h>
+#include <math.h>
+
+void TestCases ();
+
+void TestCases () {
+   ComplexNumber num1[] = { {3, 2}, {5, 7}, {1, -1} }, num2[] = { {1, -4}, {3, -8}, {-2, 3} };
+   int numOfTestCases = sizeof (num1) / sizeof (num1[0]);
+   for (int i = 0; i < numOfTestCases; i++) {
+      printf ("\nTestcase %d:\n\n", i + 1);
+      // Addition TestCase
+      ComplexNumber expectedAddOut = { num1[i].Real + num2[i].Real,num1[i].Img + num2[i].Img },
+         addRes = Op_Add (num1[i], num2[i]);
+      printf ("Addition: %s\n", IsEqual (addRes, expectedAddOut) ? "Pass" : "Fail");

```

Comment:
```suggestion
      printf ("Addition: %s\n", IsEqual (Op_Add (a, b), expectedAddOut) ? "Pass" : "Fail");

```
I think C allows you to do this.